mobile/src/full/java/org/openhab/habdroid/core/FcmMessageListenerService.kt

```diff
@@ -42,15 +45,23 @@ class FcmMessageListenerService : FirebaseMessagingService() {
 
         when (messageType) {
             "notification" -> {
+                val actions = data["actions"]
+                    ?.toJsonArrayOrNull()
+                    ?.map { it.toCloudNotificationAction() }
+                    ?.filterNotNull()
                 val cloudNotification = CloudNotification(
-                    data["persistedId"].orEmpty(),
-                    data["message"].orEmpty(),
+                    id = data["persistedId"].orEmpty(),
+                    title = data["title"].orEmpty(),
+                    message = data["message"].orEmpty(),
                     // Older versions of openhab-cloud didn't send the notification generation
                     // timestamp, so use the (undocumented) google.sent_time as a time reference
                     // in that case. If that also isn't present, don't show time at all.
-                    data["timestamp"]?.toLong() ?: message.sentTime,
-                    data["icon"].toOH2IconResource(),
-                    data["severity"]
+                    createdTimestamp = data["timestamp"]?.toLongOrNull() ?: message.sentTime,
```

Comment:
While I understand it's a good thing to not crash the app on wrong timestamp formats, we probably wouldn't have noticed the openhab-cloud issue that way...

Reply:
Maybe remove the fallback to `message.sentTime`? Then make sure that the timestamp is hidden from the notification it it's `null`.